docs/release-notes/release-notes-0.6.3.md

```diff
@@ -0,0 +1,221 @@
+
+`6b41b8b` Disable creation of fidelity bond wallets
+
+`869ef55` Disable loading of fidelity bond wallets by Qt
+
+`14f086b` Add usage guide for fidelity bond wallets
+
+`2860c4f` Freeze timelocked UTXOs with locktimes in future
+
+`ddb32ce` Rename functions to say "key" instead of "privkey"
+
+`c70183b` Create tests for fidelity bond wallets
+
+`a0a0d28` Add support for spending timelocked UTXOs
+
+`762b1f6` Add watch only wallets for fidelity bonds
+
+`a937c44` Add wallet-tool addtxoutproof method
+
+`97216d3` Sync burner outputs and display in wallet-tool
+
+`255d155` Add merkle proof functions to BitcoinCoreInterface
+
+`2271ce0` Add support for burning coins with sendpayment
+
+`dc715c9` Add timelock fidelity bond wallet sync and display
+
+`ee70cd7` Add support for OP_CLTV timelock addresses
+
+`d86df33` Rename functions which create multisig scripts
+
+`53b056e` Rename variable internal to address_type
+
+### Qt improvements and bugfixes
+
+Most important here is the new amount widget that allows entry of bitcoin amounts in sats or BTC in an intuitive way.
+
+`b1ee2f0` Make BitcoinAmountEdit behave the same on resize as QLineEdit
+
+`d1323f6` BitcoinAmountEdit Qt widget
+
+`48f5ddf` QValidator port number restriction only for ports
+
+`d8fef95` Bugfix: recovered wallet file location
+
+`3cd54cb` Adds JMIntValidator to ensure exact integer range
+
+`6f90e30` Fix "Show seed" in Qt for wallets with mnemonic extension
+
+`e536eb1` Add wrapper shell script for JoinMarketQt
+
+`554ced0` Update all exit() calls to sys.exit()
+
+`cfc4869` Install XDG desktop entry for JoinMarketQt
+
+### Fix commitments recording in jmdaemon
+
+For some time this bug resulted in bots silently ignoring commitments being broadcast in public by other bots (via `!hp2` messages). Whilst
+this did not impact coinjoin functioning, it may have erroneously allowed more usages of PoDLE commitments than intended. Removing the reference to `jm_single()` fixes this.
+
+`6d87322` Remove jm_single config var call from jmdaemon
+
+### CLI improvements and bugfixes
+
+This includes a change in default minimum size for yieldgenerators, the ability to query software version with the `--version` flag, and an occasionally important improvement in tumbler behaviour: sometimes attempts to broadcast transactions fail due to 'mempool conflict' (one of the inputs provided by a maker, is already spent). But the algorithm was treating this case very wrongly: it was assuming that the cause of the stalling was that some counterparty responded wrongly (e.g. not sending a signature), and so was re-attempting the transaction with all counterparties who *did* respond; this basically ensured repeated failure, because whatever input was already spent, is of course, always already spent. Instead after this fix, the bot will try again with new counterparties.
```

Comment:
`This includes a change in default minimum size`
This includes a change in the default minimum size?

Reply:
Think it's fine either way.